src/Microsoft.EntityFrameworkCore.Tools/tools/EntityFrameworkCore.psm1

```diff
@@ -691,6 +691,8 @@ function GetCsprojArguments($startupProject, $outputFileName) {
         $dataDirectory = Join-Path $startupProjectDir 'App_Data'
     } elseif ($appConfig) {
         $configurationFile = GetProperty $appConfig.Properties FullPath
+    } elseif (IsUwpProject $startupProject) {
+        $configurationFile = Join-Path $PSScriptRoot 'uap10.0/ef.exe.config'
```

Comment:
Note, these can be overridden by adding your own `App.config` to the startup project